torch/cuda/__init__.py

```diff
@@ -693,19 +702,55 @@ def utilization(device: Optional[Union[Device, int]] = None) -> int:
     Warning: Each sample period may be between 1 second and 1/6 second,
     depending on the product being queried.
     """
-    try:
-        import pynvml  # type: ignore[import]
-    except ModuleNotFoundError as e:
-        raise ModuleNotFoundError("pynvml module not found, please install pynvml") from e
-    from pynvml import NVMLError_DriverNotLoaded
-    try:
-        pynvml.nvmlInit()
-    except NVMLError_DriverNotLoaded as e:
-        raise RuntimeError("cuda driver can't be loaded, is cuda enabled?") from e
-    device = _get_device_index(device, optional=True)
-    handle = pynvml.nvmlDeviceGetHandleByIndex(device)
+    handle = _setup_pynvml(device)
     return pynvml.nvmlDeviceGetUtilizationRates(handle).gpu
 
+def temperature(device: Optional[Union[Device, int]] = None) -> int:
+    r"""Returns the average temperature of the GPU sensor over the past sample period as given by `nvidia-smi`.
```

Comment:
Same here, we could copy paste `Retrieves the current temperature readings for the device, in degrees C.`